CHANGELOG.md

```diff
@@ -1,6 +1,18 @@
+## 2.6.19 (November 23, 2020)
+
+* Separate connections for consuming and publishing
+* Cosuming is done with polling instead of pushing
+* Reconnects on connection errors
+* Hadling cosumer cancel notification
+* Lowered log levels of some developers' log messages
+* Addded env vars:
+    * AMQP_RECONNECT_ATTEMPTS - number of retries on connection close
+    * AMQP_RECONNECT_TIMEOUT - deplay between connection retries
+    * WAIT_MESSAGES_TIMEOUT - delay betwee next poll when queue is empty
```

Comment:
* Separate connections for consuming and publishing
* Consuming is done with polling instead of pushing
* Reconnects on connection errors
* Handling consumer cancel notification
* Lowered log levels of some developer's log messages
* Added env vars:
    * AMQP_RECONNECT_ATTEMPTS - number of retries on connection close
    * AMQP_RECONNECT_TIMEOUT - delay between connection retries
    * WAIT_MESSAGES_TIMEOUT - delay between next poll when a queue is empty